tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Test
+  block:
+    - name: Add Resource Pool
+      ansible.builtin.include_tasks: call_resource_pool_role.yml
+      vars:
+        required_state: present
+      tags: eco-vcenter-ci
+
+    - name: Provision Virtual ESXI and set ESXI hostname
+      ansible.builtin.include_tasks: provision_esxi.yml
+      tags: eco-vcenter-ci
+
+    - name: Add ESXI Host
+      ansible.builtin.import_role:
+        name: cloud.vmware_ops.vcenter_host_connection
+      vars:
+        vcenter_host_connection_state: present
```

Comment:
I would add a task to check that esxi host has been added, and a similar one after the removal to check that it is removed

Reply:
done